docs/02.API-REFERENCE.md

```diff
@@ -1710,29 +1710,34 @@ jerry_get_error_type (const jerry_value_t value);
 - [jerry_create_error](#jerry_create_error)
 - [jerry_value_is_error](#jerry_value_is_error)
 
-## jerry_value_clear_error_flag
+## jerry_get_value_from_error
 
 **Summary**
 
-Clear both the error and abort flags.
+Get the value from an error.
+
+Extract the api value from an error. If the second argument is true
+it will release the input error value.
+
+*Note*: Returned value must be freed with [jerry_release_value](#jerry_release_value) when it
+is no longer needed.
 
 **Prototype**
 
 ```c
-void
-jerry_value_clear_error_flag (jerry_value_t *value_p);
+jerry_value_t
+jerry_get_value_from_error (jerry_value_t value, bool release)
 ```
 
-- `value_p` - pointer to an api value
```

Comment:
- `value` - error (api) value
- `release` - defines whether input api value must be released